tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -1131,3 +1135,168 @@ def test_positive_provision_rhev_with_host_group(self):
+                session,
+                name=cr_name,
+                provider_type=FOREMAN_PROVIDERS['rhev'],
+                parameter_list=parameter_list,
+                orgs=[self.org_name],
+                org_select=True,
+                locations=[self.loc_name],
+                loc_select=True
+            )
+            parameter_list_img = [
+                ['Name', image_name],
+                ['Operatingsystem', os.title],
+                ['Architecture', arch.name],
+                ['Username', self.rhev_img_user],
+                ['Password', self.rhev_img_pass],
+                ['uuid', self.rhev_img_name],
+            ]
+            self.compute_resource.add_image(cr_name, parameter_list_img)
+            self.assertIn(
+                image_name,
+                self.compute_resource.list_images(cr_name)
+            )
+            self.compute_resource.set_profile_values(
+                cr_name, COMPUTE_PROFILE_SMALL,
+                cluster=self.rhev_datacenter,
+                cores=1,
+                memory=1024,
+                network_interfaces=[
+                    dict(
+                        name='nic1',
+                        network=settings.vlan_networking.bridge,
+                    ),
+                ],
+                storage=[
+                    dict(
+                        size='10',
+                        storage_domain=self.rhev_storage_domain,
+                        bootable=True,
+                        preallocate_disk=True,
+                    ),
+                ]
+            )
+            host_parameters_list = [
+                    ['Host', 'Organization', self.org_name],
+                    ['Host', 'Location', self.loc_name],
+                    ['Host', 'Host group', host_group.name],
+                    ['Host', 'Deploy on', cr_resource],
+                    ['Host', 'Compute profile', COMPUTE_PROFILE_SMALL],
+                    ['Operating System', 'Operating System', os.title],
+                    ['Operating System', 'Root password', root_pwd],
+                ]
+            make_host(
+                session,
+                name=host_name,
+                org=self.org_name,
+                loc=self.loc_name,
+                parameters_list=host_parameters_list,
+                provisioning_method='image',
+            )
+            vm_host_name = '{0}.{1}'.format(
+                host_name.lower(), domain.name)
+            # the provisioning take some time to finish, when done will be
+            # redirected to the created host
+            # wait until redirected to host page
+            self.assertIsNotNone(
+                session.hosts.wait_until_element(
+                    locators["host.host_page_title"] % vm_host_name,
+                    timeout=60
+                )
+            )
+            # Query RHEV hypervisor for vm description
+            vm_host_data = entities.Host().search_json(
```

Comment:
any necessity to use direct `search_json`. Such approach is strongly discouraged

Reply:
we need the uuid field that is returned only by read_json, read_json is a public entity function

Reply:
do we have bug for that behavior?

Reply:
You should probably add `uuid` field instead. Pls refer to https://github.com/SatelliteQE/robottelo/pull/4928#discussion_r126986809 to remind read/read_json difference.